mslib/msui/mscolab_project.py

```diff
@@ -231,14 +231,12 @@ def toggle_preview(self):
             self.previewBtn.setText("Write")
 
     def handle_upload(self):
-        img_type = "Image (*.png *.gif *.jpg *jpeg *.bmp)"
-        doc_type = "Document (*.*)"
-        file_filter = f'{img_type};;{doc_type}'
-        file_path, file_type = QtWidgets.QFileDialog.getOpenFileName(self, "Select a file", "", file_filter)
-        if file_path == "":
+        file_filter = ["Image (*.png *.gif *.jpg *jpeg *.bmp)", "Document (*.*)"]
```

Comment:
All these examples use ";;" check this with the docs what is the difference to a ","

https://stackoverflow.com/questions/1604440/how-to-set-selected-filter-on-qfiledialog

Reply:
the file_filter required for QtWidgets.QFileDialog.getOpenFileName is in the format that involves separating filters with ';;' (https://doc.qt.io/archives/qt-4.8/qfiledialog.html#getOpenFileName), but fs.file_picker takes file_filter as a list, that is why the new file_filter is a list of filters.

Reply:
there was an error with qt filepicker after the changes. I have fixed them now for both fs and qt.